genie/input_to_database.py

```diff
@@ -299,21 +298,9 @@ def validatefile(fileinfo,
 
     filetype = validate.determine_filetype(syn, filepaths, center)
     if check_file_status['to_validate']:
-        try:
-            valid, message, filetype = validate.validate_single_file(
-                syn,
-                filepaths,
-                center,
-                oncotreelink=oncotree_link,
-                testing=testing)
-            logger.info("VALIDATION COMPLETE")
-        except ValueError as e:
-            # Specify this as None for the single case where filename
-            # validation fails
-            filetype = None
-            message = str(e)
-            logger.error(message)
-            valid = False
+        valid, message, filetype = _check_valid(syn, filepaths, center, 
```

Comment:
Wrong order of returned parameters.

```suggestion
        valid, filetype, message = _check_valid(syn, filepaths, center, 
```